pkgs/applications/networking/cluster/helm/plugins/helm-s3.nix

```diff
@@ -31,7 +31,7 @@ buildGoModule rec {
   meta = with lib; {
     description = "A Helm plugin that shows a diff";
     inherit (src.meta) homepage;
-    license = licenses.apsl20;
+    license = licenses.mit;
```

Comment:
This one is actually neither apsl20 nor asl20
See https://github.com/hypnoglow/helm-s3/blob/v0.10.0/LICENSE